Formula/k/kor.rb

```diff
@@ -0,0 +1,38 @@
+class Kor < Formula
+  desc "CLI tool to discover unused Kubernetes resources"
+  homepage "https://github.com/yonahd/kor"
+  url "https://github.com/yonahd/kor/archive/refs/tags/v0.1.8.tar.gz"
+  sha256 "645f6076129fb40eb05baff8a6dcbecf29a5b09e9910779a5fabb0d11d9e6151"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w")
+  end
+
+  test do
+    (testpath/"mock-kubeconfig").write <<~EOS
+      apiVersion: v1
+      clusters:
+        - cluster:
+            server: https://mock-server:6443
+          name: mock-server:6443
+      contexts:
+        - context:
+            cluster: mock-server:6443
+            namespace: default
+            user: mockUser/mock-server:6443
+          name: default/mock-server:6443/mockUser
+      current-context: default/mock-server:6443/mockUser
+      kind: Config
+      preferences: {}
+      users:
+        - name: kube:admin/mock-server:6443
+          user:
+            token: sha256~QTYGVumELfyzLS9H9gOiDhVA2B1VnlsNaRsiztOnae0#{"    "}
```

Comment:
Any reason why these 4 spaces are there?

```suggestion
            token: sha256~QTYGVumELfyzLS9H9gOiDhVA2B1VnlsNaRsiztOnae0
```